### PR TITLE
final(ly) last configuration of legacy market

### PR DESCRIPTION
**Pull request by dbeal-eth**
this is the one to put it on mainnet!

### CODE REVIEW

File: omnibus-mainnet.toml [hunk 1]
```diff
@@ -64,3 +84,40 @@ defaultValue = "0xE561d5E02207fb5eB32cca20a699E0d8919a1476"
...
+
+[setting.synthUsdeSkewScale]
+defaultValue = "0"
+
+[setting.synth_usde_max_market_collateral]
+defaultValue = "<%= parseEther('100000000') %>"
+
++# we want to enable the spot market wrapper for usde
+[import.usde]
+source = "usde-token"
```
Comment (noisekit) [inline omnibus-mainnet.toml:99]:
```suggestion
[import.usde]
source = "usde-token"
description = "we want to enable the spot market wrapper for usde"
```

Also do we need `target` by any chance?

File: omnibus-mainnet.toml [hunk 2]
```diff
@@ -33,7 +41,16 @@ defaultValue = "0x33303637343430323300000000000000000000000000000000000000000000
 defaultValue = "snax"
 
 [setting.snx_address]
-defaultValue = "0xc011a73ee8576fb46f5e1c5751ca3b9fe0af2a6f"
+defaultValue = "<%= imports.legacyMarket.imports.v2x.contracts.ProxySynthetix.address %>"
+
+[setting.wsteth_aggregator_address]
+defaultValue = ""
```
Comment (noisekit) [inline omnibus-mainnet.toml:47]: value?

File: omnibus-mainnet.toml [hunk 3]
```diff
@@ -6,22 +6,30 @@ include = [
...
+    "tomls/markets/legacy.toml",
+		"tomls/markets/spot-factory.toml",
+		"tomls/markets/spot/usde.toml",
     "tomls/pools/spartan-council.toml",
     "tomls/permit-all-transferCrossChain.toml",
 ]
 
 [setting.snx_package]
-defaultValue = "synthetix:3.3.5"
+defaultValue = "synthetix:3.3.15"
```
Comment (noisekit) [inline omnibus-mainnet.toml:23]:
```suggestion
defaultValue = "synthetix:3.3.18"
```

File: omnibus-mainnet.toml [hunk 4]
```diff
@@ -6,22 +6,30 @@ include = [
...
 
 [setting.spot_market_package]
-defaultValue = "synthetix-spot-market:3.3.5"
+defaultValue = "synthetix-spot-market:3.3.15"
 
 [setting.perps_market_package]
-defaultValue = "synthetix-perps-market:3.3.5"
+defaultValue = "synthetix-perps-market:3.3.15"
```
Comment (noisekit) [inline omnibus-mainnet.toml:29]:
```suggestion
defaultValue = "synthetix-perps-market:3.3.19"
```

File: omnibus-mainnet.toml (hunk 1 above)
Comment (dbeal-eth) [reply omnibus-mainnet.toml:99]: target is only used on `provision` steps so no, we do not need `target`

File: omnibus-mainnet.toml (hunk 2 above)
Comment (dbeal-eth) [reply omnibus-mainnet.toml:47]: thats why this PR is still draft...

**Review by kaleb-keny — APPROVED** [review]

**Comment by dbeal-eth** [comment]
we are merging because:
* mainnet-andromeda failure seems like it cannot possibly be related to this change
* after disucsison with kaleb, th eissues wrt the test failures on sepolia can be resolved once we execute the release (not so easily before)

**Pull request merged**